src/Neo.Cryptography.BLS12_381/ConstantTimeUtility.cs

```diff
@@ -12,31 +12,32 @@
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-namespace Neo.Cryptography.BLS12_381;
-
-public static class ConstantTimeUtility
+namespace Neo.Cryptography.BLS12_381
 {
-    public static bool ConstantTimeEq<T>(in T a, in T b) where T : unmanaged
+    public static class ConstantTimeUtility
```

Comment:
Please focus in the PR changes, hard to review like this

Reply:
Its like adding the `{` so everything is wrapped with `{` `}`